mmcv/transforms/__init__.py

```diff
@@ -1,5 +1,21 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 from .builder import TRANSFORMS
-from .wrappers import ApplyToMultiple, Compose, RandomChoice, Remap
 
-__all__ = ['TRANSFORMS', 'ApplyToMultiple', 'Compose', 'RandomChoice', 'Remap']
+try:
```

Comment:
in transforms.__init__ we do not need to handle the existence of torch. We handle it in the module.